### ISSUE TITLE
Clarify number of iterations on benchmark graphs

### ISSUE THREAD

**Issue by kasperl** [harrycheung/Mobile-App-Performance#18]
It would be great if you clarify if the beautiful graphs here:

  https://medium.com/@harrycheung/mobile-app-performance-redux-e512be94f976#.1v6ezq7su

are based on 1,000 or 10,000 iterations runs. Maybe I missed it somewhere, but it's not 100% obvious to me since the source code for the different implementations support both variants.

**Comment by kasperl** [comment]
Ah, the description in the original article makes it a bit clearer I think. You're reporting the performance for 1,000 iterations and only using the 10,000 iterations version to look at memory performance, right?

**Comment by harrycheung** [comment]
Yup, you are correct. Sorry it wasn't clear. I'm not that prolific and
writing is not my forte.

**Comment by Rimobul** [comment]
Is there another/newer report about the performance outputs? One, that would cover also JavaScript, Unity and the other projects in this repo?
